src/constants/models.ts

```
@@ -101,17 +168,69 @@ export const FIXTURE_CONFIG: Record<ComponentType, FixtureConfig> = {
   // These will use your existing procedural code
-  Bath: PROCEDURAL_FIXTURES.find(f => f.name === 'Bath')!,
+
   Shower: PROCEDURAL_FIXTURES.find(f => f.name === 'Shower')!,
-  Mirror: PROCEDURAL_FIXTURES.find(f => f.name === 'Mirror')!,
+  Mirror: PROCEDURAL_FIXTURES.find(f => f.name === 'Mirror')!
+};
+
+// Helper function to get wall buffer for an object
+export const getObjectWallBuffer = (
+  objectType: ComponentType,
+  scale: number = 1.0
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)